nixarr/jellyseerr/jellyseerr-module/default.nix

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
Why create our own jellyseer module rather than using the Nixpkgs one as we do in main?